olive/passes/pytorch/qlora.py

```diff
@@ -122,6 +122,8 @@ class QLoRA(Pass):
     This pass only supports PyTorchModel with hf_config.
     """
 
+    model_overwrites: ClassVar[tuple] = ("torch_dtype", "device_map", "quantization_method", "quantization_config")
```

Comment:
It would be better to add a few comments to introduce this new field.

Reply:
Done!